tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/observability/metrics/WebSocketMetricsTestCase.java

```diff
@@ -82,7 +86,9 @@ private void setup() throws Exception {
      */
     @Test
     public void testMetrics() throws Exception {
-        WebSocketTestClient client = new WebSocketTestClient("ws://localhost:9090/basic/ws");
+        WebSocketTestClient client = new WebSocketTestClient(SERVER_URL);
+        CountDownLatch countDownLatch = new CountDownLatch(1);
+        countDownLatch.await(10, TimeUnit.SECONDS);
```

Comment:
No need to create the countDownLatch to handshake because we already set it in the implementation. Please refer the [test](https://github.com/ballerina-platform/ballerina-lang/blob/master/tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/service/websocket/SimpleProxyTestCommons.java#L65).